### ISSUE TITLE
failure to find memkind

### ISSUE THREAD

**Issue by kilobyte** [pmem/pmemkv#487]
`dpkg -l libmemkind-dev` says `1.9.0-2`, yet upon trying to build:
```
-- Checking for module 'libpmemobj++>=1.8'
--   Found libpmemobj++, version 1.8-rc1
-- Checking for module 'memkind'
--   Found memkind, version 1.9.0
-- Performing Test LIBMEMKIND_NAMESPACE_PRESENT
-- Performing Test LIBMEMKIND_NAMESPACE_PRESENT - Failed
-- libmemkind namespace not found (available in memkind > 1.9.0). Old namespace will be used for 'pmem::allocator'.
```

**Comment by igchor** [comment]
Warning message should be probably written differently. What it's trying to say is that libpmemkind namespace will be available in the minor version bigger than 9 (that is 1.10.0)

**Comment by kilobyte** [comment]
It's on memkind master, thus that'll be 1.10, yeah (or 2.0, but that matches >= relation).

**Comment by lukaszstolarczuk** [comment]
some fixes also delivered in #489
